src/test/java/api/loans/DeclareLostAPITests.java

```diff
@@ -147,7 +147,7 @@ public void shouldChargeProcessingAndItemFeesWhenBothDefined() {
       .withName("Declared lost fee test policy")
       .chargeProcessingFee()
       .withLostItemProcessingFee(expectedProcessingFee)
-      .withChargeAmountItem("anotherCost", expectedItemFee);
+      .withSetCostCharge(expectedItemFee);
```

Comment:
Can you help me understand where the language of charge comes from?

I haven't managed to find a way disambiguate the fee defined in the policy and the fee charged to a patron, is charge intended to do that?

Reply:
The node with item fee configuration for lost item policy is called `chargeAmountItem`.

Reply:
I think the builders (like the domain) in mod-circulation should try to hide the details of the modelling in fees and fines (which I have reservations about).

Could this be `withSetCost` instead?